iotdb-core/node-commons/src/main/java/org/apache/iotdb/commons/pipe/config/constant/PipeProcessorConstant.java

```diff
@@ -77,6 +87,22 @@ public class PipeProcessorConstant {
   public static final long PROCESSOR_CHANGING_VALUE_MAX_TIME_INTERVAL_DEFAULT_VALUE =
       Long.MAX_VALUE;
 
+  public static final String PROCESSOR_CHANGING_POINT_ARRIVAL_TIME_MIN_INTERVAL =
+      "processor.changing-point.arrival-time.min-interval";
+  public static final String PROCESSOR_CHANGING_POINT_ARRIVAL_TIME_MAX_INTERVAL =
+      "processor.changing-point.arrival-time.max-interval";
+  public static final String PROCESSOR_CHANGING_POINT_EVENT_TIME_MIN_INTERVAL =
+      "processor.changing-point.event-time.min-interval";
```

Comment:
PTAL at the feishu doc? Maybe the "changing-point" and the "sdt" prefix is not here for "arrival-time" and "event-time"... And the customization (currently validation) shall be placed in the super class "DownSamplingProcessor"?

Reply:
DownSamplingProcessor subclasses do not necessarily support arrivalTime. . . . .